### DISCUSSION TITLE
(Projects Beta) "Cannot add items when grouped by milestone"

### DISCUSSION THREAD

**Discussion by lisa-wolfgang** [community/community#9575] [Projects and Issues]
Grouping by milestone is a very useful way to view a comprehensive timeline of a project, and creating issues directly from the project board is very convenient, but we unfortunately cannot have both at the same time.

I'd imagine this has to be a technical problem. Currently, our workaround is to just create the issues in another view and then switch back to the main view, but it would be really nice if we could create issues while staying in the main view. It would be even better if the group where the issue is created would automatically have that milestone applied to it.

**Comment by nicbet** [comment]
We just started the process of moving off of JIRA to GitHub and the first thing we tried with the new Projects beta was to setup a planning spreadsheet, with issues grouped by milestone (a set of quarterly release milestones, and a catch-all milestone called `Treasure Chest` that tracks all items not yet evaluated and allocated to a release). Not being able to add items when `group by milestone` is active, is quite irritating.

*Edit*:
Interestingly, the marketing page for GH Projects shows this functionality available and working, so I'm wondering whether I'm doing anything wrong in the table view?

![Screen Shot 2022-02-24 at 10 57 24 AM](https://user-images.githubusercontent.com/718340/155560151-911aab49-4f4e-4839-9962-89ff8de27ddb.png)

**Reply by nicbet** [reply]
*Update*:

I think I've figured this out ... looking at the marketing example, they are not using the (builtin) `milestone` column, but created a `single select` type column called `milestones`, with option values according to the release milestones. When grouping by this new single-select type column called `miletstones`, the spreadsheet shows an `add item` entry field as expected.

**Reply by TheNewJavaman** [reply]
Sneaks pretty sneaky if I'm being honest.

**Reply by naedx** [reply]
I'm also interested in this. It works EXACTLY as expected if I were to create my own "Milestones" field as described in @nicbet's workaround.

Having to work around it feels unnecessary and inefficient.

**Comment by MrSkwiggs** [comment]
This is a pretty common operation (grouping by milestone, and adding an item to a given milestone). Would be great to have 🙏

**Comment by naedx** [comment]
Is there any news on this feature?

**Comment by bradsk88** [comment]
Every day I'm haunted 😓 

![image](https://github.com/community/community/assets/2147630/a5bb0ba0-f4fb-4a18-8333-319d3ee42b9f)

**Comment by Apolexian** [comment]
Still?
